### PR TITLE
Migrate content changes from `polaris-react/**/README.md` files to polaris.shopify.com

### PR DESCRIPTION
**Pull request by alex-page**
- [x] Add missing content
- [x] Enforce reviews for `README.md` files

### CODE REVIEW

File: polaris.shopify.com/content/components/modal.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Use modals for confirmations and conditional changes. They should be thought of
 Modals should:
 
 - Require that merchants take an action.
-- Close when merchants press the `X` button, the `Cancel` button, or the <kbd>Esc</kbd> key, not when merchants click or tap the area outside the modal.
+- Close when merchants press the `X` button, the `Cancel` button, the <kbd>Esc</kbd> key, or when merchants click or tap the area outside the modal.
```
Comment (martenbjork) [inline polaris.shopify.com/content/components/modal.md:70]: Nice one!

File: polaris.shopify.com/content/foundations/patterns/error-messages.md [hunk 2]
```diff
@@ -300,11 +300,7 @@ Once a field has an error, complete validation checks after each keystroke.
 Remove the error message as soon as the input becomes valid so merchants can
 immediately tell they fixed the issue.
 
-![Text field validation with purple border](/images/foundations/patterns/error-messages/text-field-validation-purple-border-cursor@2x.png)
-
-If the validation process is less than a second but not instant, show a spinner
 on the field to indicate validation progress.
-
```
Comment (martenbjork) [inline polaris.shopify.com/content/foundations/patterns/error-messages.md]: This stuff looks broken. It only renders this:

> on the field to indicate validation progress.

**Review by martenbjork — CHANGES_REQUESTED** [review]

File: polaris.shopify.com/content/foundations/patterns/error-messages.md (hunk 2 above)
Comment (alex-page) [reply polaris.shopify.com/content/foundations/patterns/error-messages.md]: Yep. Fixed this up. Silly vscode.

File: polaris-react/src/components/ChoiceList/README.md [hunk 3]
```diff
@@ -85,9 +85,15 @@ If the customer abandons their checkout, send them an email reminder to complete
 - Option a
 - Option b
 
+- Option a
+- Option b
+
```
Comment (alex-page) [inline polaris-react/src/components/ChoiceList/README.md]:
```suggestion
```

File: polaris-react/src/components/ChoiceList/README.md [hunk 4]
```diff
@@ -87,7 +87,10 @@ If the customer abandons their checkout, send them an email reminder to complete
 
 #### Don’t
 
-If the customer abandons their checkout, send them an email reminder to complete their order
+If the customer abandons their checkout, send them an email reminder to complete their order:
+
+- Option a
+- Option b
```
Comment (alex-page) [inline polaris-react/src/components/ChoiceList/README.md]:
```suggestion
If the customer abandons their checkout, send them an email reminder to complete their order
```

**Pull request merged**